packages/playground/src/index.css

```
@@ -3,6 +3,8 @@
 /* Scan playground-ui package for Tailwind classes */
 @source '../../node_modules/@mastra/playground-ui';
 
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.7.0)

[error] Expected no empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)